src/Hide/_tests_/index.test.js

```diff
@@ -35,6 +37,12 @@ describe("Hide", () => {
     );
     expect(component).toHaveStyleRule("display", "none");
   });
+  it("should be displayed block", () => {
+    component.setProps({ block: true, on: breakpoints[on[1]] });
```

Comment:
If you want to do it this way, should be
```
component.setProps({ block: true, on: on[1] });
```

Otherwise, you are setting a value of `(min-width: something)` not e.g. `smallMobile`.

You can also create a new describe if you want to.